builtin/mainmenu/pkgmgr.lua

```diff
@@ -682,10 +682,9 @@ function pkgmgr.preparemodlist(data)
 	local game_mods = {}
 
 	--read global mods
-	local modpath = core.get_modpath()
-
-	if modpath ~= nil and
-		modpath ~= "" then
+	local modpaths = core.get_modpaths()
+	--XXX what was ‘modpath ~= ""’ and ‘modpath ~= nil’ for?
```

Comment:
it's fine now in either case since `get_modpaths` can return an empty list
```suggestion
```

Reply:
I'll remove the XXX in a revised patch